File: Serenity Practice/HosbitalSerenity.Web/Modules/Default/Governorate/GovernorateColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace HosbitalSerenity.Default.Columns;

[ColumnsScript("Default.Governorate")]
[BasedOnRow(typeof(GovernorateRow), CheckNames = true)]
public class GovernorateColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int GovId { get; set; }
    [EditLink]
    public string GovName { get; set; }
}

File: Serenity Practice/HosbitalSerenity.Web/Modules/ServerTypes/Default/GovernorateColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { GovernorateRow } from "./GovernorateRow";

export interface GovernorateColumns {
    GovId: Column<GovernorateRow>;
    GovName: Column<GovernorateRow>;
}

export class GovernorateColumns extends ColumnsBase<GovernorateRow> {
    static readonly columnsKey = 'Default.Governorate';
    static readonly Fields = fieldsProxy<GovernorateColumns>();
}